contrib/systemd/labgrid-exporter.service

```diff
@@ -8,9 +8,9 @@ Environment="PYTHONUNBUFFERED=1"
 # Should contain LG_COORDINATOR configuration
 EnvironmentFile=-/etc/environment
 ExecStart=/path/to/labgrid/venv/bin/labgrid-exporter /etc/labgrid/exporter.yaml
-Restart=on-failure
-RestartForceExitStatus=100
+Restart=always
 RestartSec=30
+StartLimitIntervalSec=0
```

Comment:
Is this a valid option?  I am getting the following in my log files:
```
Feb 25 13:35:52 testlab02 systemd[1]: /etc/systemd/system/labgrid-exporter.service:13: Unknown key 'StartLimitIntervalSec' in section [Service], ignoring.
```

```
(labgrid-venv) pi@testlab02:~/labgrid $ uname -a
Linux testlab02 6.6.74+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.74-1+rpt1 (2025-01-27) aarch64 GNU/Linux
(labgrid-venv) pi@testlab02:~/labgrid $ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 12 (bookworm)
Release:	12
Codename:	bookworm
```

Reply:
Hmm, the StartLimit options need to go in the [Unit] section.

Reply:
I'll create a PR.

Reply:
See #1620.